src/Fields/NodeIdResolver.php

```diff
@@ -7,33 +7,38 @@
 class NodeIdResolver
 {
 
-    const INDEX_TYPE = 0;
-    const INDEX_ID   = 1;
+    private const INDEX_TYPE = 0;
+    private const INDEX_ID   = 1;
 
     /**
      * @param string $globalId
+     *
      * @return string
+     * @throws MalformedNodeIdException
      */
-    public static function typeFromGlobalId($globalId)
+    public static function typeFromGlobalId($globalId): string
```

Comment:
Should we type-hint for the parameter(s) also?

Reply:
Sure